### PR TITLE
Kevin/eu content updates

### PR DESCRIPTION
**Pull request by kevinw-binary**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/dmt5/_interest_free_trading.js [hunk 1]
```diff
@@ -93,40 +148,35 @@ CheckedText.propTypes = {
...
-        </Show.Desktop>
+        <StyledContainer>
+            <InterestCardFlex fd="column" ai="center" jc="center">
+                <ImageWrapper>
+                    <img src={ZeroPercent} alt="Zero percent interest" width="100%" height="100%" />
+                </ImageWrapper>
+                <InterestText size="32px">
+                    <Localize translate_text="interest" />
+                </InterestText>
+                <TextWrapperFlex fd="column" jc="flex-start" ai="flex-start">
```
Comment (habib-binary) [inline src/pages/dmt5/_interest_free_trading.js:159]: Do we need Localization for these ?

Comment (kevinw-binary) [reply src/pages/dmt5/_interest_free_trading.js:159]: I guess yeah, for this part

**Review by habib-binary — APPROVED** [review]
LGTM

File: src/pages/dmt5/_flexibility.js [hunk 2]
```diff
@@ -126,6 +127,8 @@ const StyledText = styled(Text)`
 const Flexibility = () => {
     const { is_eu_country } = React.useContext(DerivStore)
 
+    const chosenContent = is_eu_country ? eucontent : content
```
Comment (sean-binary) [inline src/pages/dmt5/_flexibility.js:130]: Let's name the variables in snake_case here.

File: src/pages/trade-types/multiplier/_how-multiplier-works.js [hunk 3]
```diff
@@ -166,21 +179,35 @@ const HowOptionsWorks = () => {
...
                         }
                     >
-                        <QueryImage
-                            data={data['multiplier_market']}
-                            alt="Trade types option market"
-                        />
+                        {is_eu_country ? (
+                            <QueryImage
+                                data={data['multiplier_market_eu']}
+                                alt="Trade types option market eu"
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_how-multiplier-works.js:185]: I don't think the alt should include the word `eu` here.

File: src/pages/trade-types/multiplier/_how-multiplier-works.js [hunk 4]
```diff
@@ -166,21 +179,35 @@ const HowOptionsWorks = () => {
...
                         }
                     >
-                        <QueryImage
-                            data={data['multiplier_trade_type']}
-                            alt="Trade types option trade type open"
-                        />
+                        {is_eu_country ? (
+                            <QueryImage
+                                data={data['multiplier_trade_type_eu']}
+                                alt="Trade types option trade type open eu"
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_how-multiplier-works.js:203]: Same here.

File: src/pages/trade-types/multiplier/_how-multiplier-works.js [hunk 5]
```diff
@@ -196,13 +223,24 @@ const HowOptionsWorks = () => {
...
                         }
                     >
-                        <QueryImage
-                            data={data['multiplier_value']}
-                            alt="Trade types multiplier value"
-                        />
+                        {is_eu_country ? (
+                            <QueryImage
+                                data={data['multiplier_value_eu']}
+                                alt="Trade types multiplier value eu"
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_how-multiplier-works.js:236]: Same here.

File: src/pages/trade-types/multiplier/_what-are-multipliers.js [hunk 6]
```diff
@@ -151,93 +173,207 @@ const WhatAreOptions = () => {
...
+                                        <QueryImage data={data['stake_amount']} alt="tes2" />
+                                    </ImgWrapper>
+                                </RowColumn>
+                            </Row>
+                            <Row>
+                                <RowColumn>
+                                    <ImgWrapper>
+                                        <QueryImage
+                                            data={data['multiplier_no_multi_win_eu']}
+                                            alt="tes"
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_what-are-multipliers.js:196]: tes? Few more instances below.

File: src/pages/dmt5/_interest_free_trading.js [hunk 7]
```diff
@@ -10,19 +10,39 @@ import ZeroPercent from 'images/svg/zero_percent.svg'
...
 
 const ImageWrapper = styled.div`
-    max-width: 124px;
-    width: 100%;
+    width: 124px;
+    height: 84px;
     object-fit: contain;
     margin-top: 4rem;
+    @media ${device.tabletL} {
+        margin-top: 6rem;
```
Comment (sean-binary) [inline src/pages/dmt5/_interest_free_trading.js:18]: 60px here?

File: src/pages/dmt5/_interest_free_trading.js [hunk 8]
```diff
@@ -10,19 +10,39 @@ import ZeroPercent from 'images/svg/zero_percent.svg'
...
     padding: 4rem;
     ${CardStyle}
+
+    @media ${device.tabletL} {
+        font-size: 14px;
+        width: 328px;
+        height: 373px;
+        margin-top: 2.4rem;
+        margin-right: 0;
+        margin-bottom: 2.4rem;
```
Comment (sean-binary) [inline src/pages/dmt5/_interest_free_trading.js:38]:
```suggestion
        margin: 24px 0;
```

**Review by sean-binary — APPROVED** [review]

**Pull request merged**